checkmate/__init__.py

```diff
@@ -0,0 +1,35 @@
+import tensorflow as tf
```

Comment:
What is the purpose of the __init__.py file? It should only be called if someone imports the TensorFlow 2.0 wrapper. Some people may use Checkmate for its solver without the TensorFlow code.

Reply:
Run any code to turn off CSE in the wrapper (https://github.com/parasj/checkmate/blob/b9f1d0c84e3bf8c71394c65c4e399e616c9d0864/checkmate/tf2/wrapper.py#L25-L32), not in init code

Reply:
Basically I tried to simplify the code in the compile_tf2 file to streamline the interface.  Its possible there are use cases where users will want to define their own gradient function, and compile_tf2 limits you in construction of the function.  By modularizing the functionality into the init.py, using the code becomes relatively easy.  My idea for this was to create a streamlined version importable from the base checkmate import that would help people use it out of the box.  You're right though;  This should be put in either tf2 or wrapper directly.